projects/js-packages/charts/src/components/pie-chart/stories/index.docs.mdx

```diff
@@ -0,0 +1,406 @@
+import { Meta, Canvas, Story, Source } from '@storybook/addon-docs/blocks';
+import * as PieChartStories from './index.stories';
+import * as DonutStories from './donut.stories';
+import * as LegendStories from './legend.stories';
+
+<Meta title="JS Packages/Charts/Types/Pie Chart" of={ PieChartStories } />
+
+# Pie Chart
+
+Pie Charts visualize proportional data as circular segments, making it easy to understand parts of a whole. They excel at showing percentage breakdowns and comparative relationships between categories.
+
+<Canvas of={ PieChartStories.Default } />
+
+## Overview
+
+The Pie Chart component provides a flexible, accessible, and highly customizable solution for displaying proportional data. Built on `@visx/shape`, it supports interactive tooltips, legends, and customizable styling with gap control, corner rounding, and thickness adjustment:
+
+<Source
+	language="jsx"
+	code={ `import { PieChart } from '@automattic/charts';
+
+	<PieChart
+		data={ data }
+		withTooltips={ true }
+		showLegend={ true }
+	/>` }
+/>
+
+## Basic Usage
+
+### Simple Pie Chart
+
+The simplest pie chart requires only a `data` prop with percentage-based data:
+
+<Canvas of={ PieChartStories.Default } />
+
+<Source
+	language="jsx"
+	code={ `<PieChart
+		size={ 400 }
+		data={[
+			{ label: 'MacOS', value: 30000, percentage: 23 },
+			{ label: 'Linux', value: 22000, percentage: 17 },
+			{ label: 'Windows', value: 80000, percentage: 60 },
+		]}
+	/>` }
+/>
+
+### Required Props
+
+- **`data`**: Array of `DataPointPercentage` objects containing label, value, and percentage
+- **`size`**: Diameter of the chart in pixels (when not using responsive mode)
+
+### Optional Props
+
+- **`padding`** (default: `20`): Padding around the chart in pixels
+- **`withTooltips`** (default: `false`): Enables interactive tooltips on hover
+- **`showLegend`** (default: `false`): Shows a legend for the chart data
+- **`gapScale`** (default: `0`): Scale of gaps between segments (0-1)
+- **`cornerScale`** (default: `0`): Scale of corner rounding for segments (0-1)
+
+## Chart Styling
+
+### Pie Chart (Default)
+
+The standard pie chart displays data as complete circular segments:
+
+<Canvas of={ PieChartStories.Default } />
+
+### Enhanced Pie Chart with Styling
+
+Pie charts can include gaps and rounded corners for modern appearance:
+
+<Canvas of={ DonutStories.Doughnut } />
+
+<Source
+	language="jsx"
+	code={ `<PieChart
+		size={ 400 }
+		gapScale={ 0.03 }
+		cornerScale={ 0.03 }
+		padding={ 0 }
+		data={ data }
+	/>` }
+/>
+
+## Interactive Features
+
+### Tooltips
+
+Enable interactive tooltips that display data information on hover:
+
+<Canvas of={ PieChartStories.WithTooltips } />
+
+<Source
+	language="jsx"
+	code={ `<PieChart
+		size={ 400 }
+		data={ data }
+		withTooltips={ true }
+	/>` }
+/>
+
+### Tooltips with Pie Chart
+
+Tooltips work seamlessly with pie chart variations:
+
+<Canvas of={ DonutStories.WithTooltipsDoughnut } />
```

Comment:
This example is also weird, I think we can remove it

Reply:
Addressed in 0a5b825dc2